server/sonar-server/src/main/java/org/sonar/server/computation/step/ComputeMeasureVariationsStep.java

```diff
@@ -100,14 +102,14 @@ public void execute() {
 
     private final DbSession session;
     private final Set<Integer> metricIds;
-    private final Map<String, Metric> metricByKeys;
+    private final Iterable<Metric> metrics;
 
-    public VariationMeasuresVisitor(DbSession session, Iterable<Metric> metrics) {
+    public VariationMeasuresVisitor(DbSession session, List<Metric> metrics) {
```

Comment:
why change `Iterable` to `List` and not store the `metrics` as a `List`. There's a inconsistency here